sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/Models/NetAppVolumeBackupConfiguration.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using Azure.Core;
+
+namespace Azure.ResourceManager.NetApp.Models
+{
+    /// <summary> Volume Backup Properties. </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public partial class NetAppVolumeBackupConfiguration
```

Comment:
Do we have its generated version previously? Why I did not see it gets renamed to this here?

Reply:
Not sure, it has not been  renamed 

https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/netapp/Azure.ResourceManager.NetApp/src/Generated/Models/NetAppVolumeBackupConfiguration.cs

Reply:
I see, this class is not removed in new version. Only this property `public ResourceIdentifier VaultId { get; set; }` is removed.
Then, we cannot add `[EditorBrowsable(EditorBrowsableState.Never)]` attribute on the class (because it is not deprecated), instead, we need to add this attribute to the property.

One question: will the model still return `vaultId` on an older api-version?
If the answer is yes, we need to move its deserialization to customization code directory, and add the deserialization of this property `vaultId` back to that method.